tasking/pulp/celery_app.py

```diff
@@ -19,10 +24,10 @@
 from pulp.tasking.constants import TASKING_CONSTANTS
 
 # This import is here so that Celery will find our application instance
-from pulp.tasking.celery_instance import celery  # noqa
+from pulp.celery_instance import celery  # noqa
 
 # This import is here so Celery will discover all tasks
-import pulp.tasking.registry  # noqa
+import pulp.tasking.registry # noqa
```

Comment:
I think 2 spaces are expected between code and an inline comment. This should likely be reverted.